src/classes/particle.ts

```diff
@@ -262,18 +268,18 @@ export class Particle {
         return res;
     }
 
-    public draw() {
+    public draw(): void {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Declaration of public instance method not allowed after declaration of private instance method. Instead, this should come after constructors.](https://app.codacy.com/manual/ar3s/tsparticles/pullRequest?prid=5011230)